stl/inc/chrono

```diff
@@ -5271,18 +5276,17 @@ namespace chrono {
         }

```

Comment:
https://github.com/microsoft/STL/pull/1812#discussion_r610053708 doesn't appear to be resolved - I see a `'}'` check being added after the `'E'`/`'O'` modifier below, but not after the `'%'` here.